api/src/main/java/io/opentelemetry/metrics/Meter.java

```diff
@@ -191,6 +191,32 @@
    */
   MeasureLong.Builder measureLongBuilder(String name);
 
+  /**
+   * Returns a new builder for a {@link MeasureDouble}.
```

Comment:
```suggestion
   * Returns a new builder for a {@link ObserverDouble}.
```

Same for all others.